diff -ruN exampleTop/configure/CONFIG_SITE binaryOpcuaIoc/configure/CONFIG_SITE
--- exampleTop/configure/CONFIG_SITE	2022-03-20 16:34:41.000000000 +0000
+++ binaryOpcuaIoc/configure/CONFIG_SITE	2023-02-15 17:56:58.263113223 +0000
@@ -59,12 +59,12 @@
 
 # Inside an embedded TOP, use the following definitions
 # pointing to the configuration of the parent module
--include $(TOP)/../../CONFIG_SITE.local
--include $(TOP)/../../configure/CONFIG_SITE.local
--include $(TOP)/../configure/CONFIG_SITE.local
--include $(TOP)/configure/CONFIG_SITE.local
-
-# Inside a standalone TOP, use the following definitions instead
-#-include $(TOP)/../CONFIG_SITE.local
+#-include $(TOP)/../../CONFIG_SITE.local
+#-include $(TOP)/../../configure/CONFIG_SITE.local
 #-include $(TOP)/../configure/CONFIG_SITE.local
 #-include $(TOP)/configure/CONFIG_SITE.local
+
+# Inside a standalone TOP, use the following definitions instead
+-include $(TOP)/../CONFIG_SITE.local
+-include $(TOP)/../configure/CONFIG_SITE.local
+-include $(TOP)/configure/CONFIG_SITE.local
diff -ruN exampleTop/configure/RELEASE binaryOpcuaIoc/configure/RELEASE
--- exampleTop/configure/RELEASE	2022-03-20 16:34:41.000000000 +0000
+++ binaryOpcuaIoc/configure/RELEASE	2023-02-15 17:57:30.009880905 +0000
@@ -41,12 +41,12 @@
 
 # Inside an embedded TOP, use the following definitions
 # pointing to the configuration of the parent module
--include $(TOP)/../../RELEASE.local
--include $(TOP)/../../configure/RELEASE.local
--include $(TOP)/../configure/RELEASE.local
--include $(TOP)/configure/RELEASE.local
-
-# Inside a standalone TOP, use the following definitions instead
-#-include $(TOP)/../RELEASE.local
+#-include $(TOP)/../../RELEASE.local
+#-include $(TOP)/../../configure/RELEASE.local
 #-include $(TOP)/../configure/RELEASE.local
 #-include $(TOP)/configure/RELEASE.local
+
+# Inside a standalone TOP, use the following definitions instead
+-include $(TOP)/../RELEASE.local
+-include $(TOP)/../configure/RELEASE.local
+-include $(TOP)/configure/RELEASE.local
